File: fs-project-react/src/models/enums/FitnessLevel.ts
export enum FitnessLevel {
    BEGINNER = 1,
    INTERMEDIATE = 2,
    ADVANCED = 3,
}

export const FitnessLevelDescriptions: { [key in FitnessLevel]: string } = {
    [FitnessLevel.BEGINNER]: "Suitable for new exercisers or those returning after a break.", // מתאים למתחילים
    [FitnessLevel.INTERMEDIATE]: "Designed for individuals with a good fitness foundation.", // מתאים לביניים
    [FitnessLevel.ADVANCED]: "For experienced exercisers aiming for maximum intensity.", // מתאים למתקדמים
};

File: java/fs-project/src/main/java/com/example/fs_project/enums/FitnessLevel.java
package com.example.fs_project.enums;

public enum FitnessLevel {
    BEGINNER(1, "Suitable for new exercisers or those returning after a break."),//מתחילים
    INTERMEDIATE(2, "Designed for individuals with a good fitness foundation."),//ביניים
    ADVANCED(3, "For experienced exercisers aiming for maximum intensity.");//מתקדמים

    private final int difficulty;
    private final String description;

    // Constructor
    FitnessLevel(int difficulty, String description) {
        this.difficulty = difficulty;
        this.description = description;
    }

    // Getters
    public int getDifficulty() {
        return difficulty;
    }

    public String getDescription() {
        return description;
    }
}
